applications/tari_app_grpc/proto/types.proto

```diff
@@ -25,6 +25,8 @@ package tari.rpc;
 
 import "google/protobuf/timestamp.proto";
 
+/// An Empty placeholder for endpoints without request parameters
+message Empty {}
```

Comment:
I'm not a fan of having an `Empty` type. It kind of goes against the forward compatibility that `protobuf` is trying to achieve. It would be better to have a `GetVersionRequest` that is empty so that fields can be added to it in future. I'll admit that it is unlikely that `GetVersionRequest` ever needs members, but I think it's simpler for new devs if there is a clear pattern that we stick to

Reply:
You can replace `Empty` with `GetVersionRequest` later. The protobuf wire-format doesn't care about the message "type" just about it's shape. Though philosophically, I agree. I'll remove the empty type here and in the other base node call where it is used.